functions/index.js

```diff
@@ -24,7 +24,7 @@ const SHEETS = {
 };
 
 if (functions.config().googleapi !== undefined) {
-  CONFIG_CLIENT_ID = functions.config().googleapi.client_id
+  CONFIG_CLIENT_ID = functions.config().googleapi.client_id;
```

Comment:
It looks like the project doesn't have Prettier or anything else installed to ensure standardized code formatting. Are folks interested in that? If so, I'm happy to add it.

Reply:
We do use eslint inside the cloud functions.

I had been initially resistant to adding formatting tools since I expected this project to sunset in ~2 weeks. However, we're still around. :)

If you want to do something to standardize our formatting, I'd be happy to approve! I'm used to eslint, but honestly have very little opinions on the specific tool.  However, it'd be nice to use the same tool everywhere.